opm/input/eclipse/Schedule/MSW/Compsegs.hpp

```diff
@@ -58,6 +58,13 @@ namespace Compsegs {
                     const ParseContext& parseContext,
                     ErrorGuard& errors);
 
+    std::pair<WellConnections, WellSegments>
+    processCOMPSEGS(std::vector<std::tuple<double, double, std::array<int, 3>>>& segments_md_and_ijk,
```

Comment:
When we use capitalized `COMPSEGS`, we are referring to the keyword `COMPSEGS`, which is not the case here. Even later we use `COMPSEGS` keyword for the grid-independent multisegment wells, we can still make it clear this function is used for grid-independent wells.  There is no need to share the same function name, which might be better for readability.

Reply:
I have renamed it.